src/oncall/api/v0/team_escalate.py

```diff
@@ -0,0 +1,30 @@
+from ... import db, iris
+from ...utils import load_json_body
+from ...auth import login_required
+from falcon import HTTPBadRequest
+from requests import ConnectionError
+
+
+@login_required
+def on_post(req, resp, team):
+    data = load_json_body(req)
+
+    connection = db.connect()
+    cursor = connection.cursor()
+    cursor.execute('SELECT iris_plan FROM team WHERE name = %s', team)
+    plan_name = cursor.fetchone()[0]
+    cursor.close()
+    connection.close()
+
+    requester = req.context.get('user')
+    if not requester:
+        requester = req.context['app']
+    data['requester'] = requester
+    if 'description' not in data:
+        raise HTTPBadRequest('Iris escalation failed', 'Escalation cannot have an empty description')
+    if plan_name is None:
+        raise HTTPBadRequest('Iris escalation failed', 'No escalation plan specified for team: %s' % team)
+    try:
+        iris.client.incident(plan_name, context=data)
+    except (ValueError, ConnectionError) as e:
+        raise HTTPBadRequest('Iris escalation failed', 'Iris client error: %s' % e)
```

Comment:
if it's returning 200 and header is json/application, then we need to return a valid json body here. or we can set the response code to 204.